### DISCUSSION TITLE
Change auth user id from uuid to bigInt, help

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#6709] [Questions]
Hi is there a guide where it show how to swap the UUID with a bigIncrement for the user id... on auth schema?
or is there a way we can change the default auth schema, when im trying to change the id from uuid to big int ...
it wont let me...

Hope someone in supabase team or any one can share their thoughts about it...

**Comment by GaryAustin1** [comment]
I'm not on the SB team, but I can't see how a change to the column type could occur.  It would affect gotrue anywhere it uses the table, storage with it's foreign key, probably realtime and any other area tokens are used,

You should be able to create your own table on insert trigger of auth.users and set up a primary incrementing bigint id and foreign key back to auth.users uuid.   Of course depending how you wanted to use it.
